src/main/java/github/grace000/erversay/RequestRouter.java

```diff
@@ -0,0 +1,22 @@
+package github.grace000.erversay;
+
+import java.util.HashMap;
+
+import static github.grace000.erversay.Constants.*;
+
+public class RequestRouter {
+    private HashMap<String, String> routeTable = new HashMap<>();
```

Comment:
You're not in Javascript anymore 😛 . You should only use a `HashMap` when you don't know the format of the data you're working with. In this case, it's your data, so you know the fields, so you should return a class with those fields, e.g.

``` java
class RoutedRequest {
  public final Request routedRequest;
  public final String responseCode;
  public final String responseStatus;

  public RoutedRequest(/* args... */) { /* implementation... */ }
}
```

This will also prevent you from having to cast `HashMap.get` results [later](https://github.com/grace000/erversay/pull/7/files#diff-2e990deebfba5e929350ec4df030b567R28).